releasenotes/notes/added-sign-feature-parameterexpression-57693dd69103dc8c.yaml

```diff
@@ -0,0 +1,22 @@
+---
+prelude: >
+    New feature to display sign of a Parameter
```

Comment:
```suggestion
```

The prelude is used for an overview of the entire release. I don't think we need to call this out there. It is enough to list it in the `features` section.